R/viz.R

```diff
@@ -2727,10 +2728,63 @@ prettyForestPlot = function(maf,
     dplyr::mutate(test = map(table, fisher.test), tidy = map(test, broom::tidy)) %>%
     unnest(tidy) %>%
     dplyr::mutate(q.value = p.adjust(p.value, "BH")) %>%
-    dplyr::select(-c(table, test, method, alternative)) %>%
+    dplyr::select(-c(test, method, alternative)) %>%
     dplyr::filter(q.value <= max_q) %>%
     dplyr::mutate(gene = fct_reorder(gene, estimate))
 
+  flatten_table <- function(Row){
```

Comment:
This functionality is added so the `fisher` result of calling  this function now also reports 4 additional columns: number of mutated and unmutated samples for each comparison group